packages/nodes-base/nodes/Postgres/Postgres.node.ts

```diff
@@ -72,10 +72,38 @@ export class Postgres implements INodeType {
 					},
 				},
 				default: '',
-				placeholder: 'SELECT id, name FROM product WHERE id < 40',
+				placeholder: 'SELECT id, name FROM product WHERE qty > $1 AND price <= $2',
 				required: true,
 				description: 'The SQL query to execute.',
 			},
+			{
+				displayName: 'Use query parameters',
+				name: 'useQueryParams',
+				type: 'boolean',
+				displayOptions: {
+					show: {
+						operation: ['executeQuery'],
+					},
+				},
+				default: false,
+				required: true,
+				description: 'Use Parametrized Queries, where variables are replaced using $1, $2, etc.. <br>Do not use this for regular n8n expressions.',
+			},
+			{
+				displayName: 'Properties',
+				name: 'properties',
+				type: 'string',
+				displayOptions: {
+					show: {
+						operation: ['executeQuery'],
+						useQueryParams: [true],
+					},
+				},
+				default: '',
+				placeholder: 'qty,price',
```

Comment:
ref previous comment
```suggestion
				placeholder: 'quantity,price',
```